trino/client.py

```diff
@@ -229,6 +231,84 @@ def __repr__(self):
             )
         )
 
+    def _col_func(self, column):
```

Comment:
Processing the `column` section (to get a list of mapping lambdas) and processing the rows can be done in multiple locations. I performed the former in the `TrinoStatus` class because this is where the data is first received. Processing the rows is done downstream, so left it there.

As far as the best candidate to perform both operations, I don't have any string opinion

Reply:
By extracting it to its own class it would allow for separation of concern of each class and put it in the most appropriate place (composition).

Reply:
Fair enough. But there is still the question about whether the location where mapping function generation and execution should take place.